src/lib.rs

```diff
@@ -132,12 +132,25 @@ pub struct Bytes32 {
     pub bytes: [u8; 32],
 }
 
+/// Alias representing a value in wei.
 pub type EtherValue = Uint128;
+
+/// Alias representing an address.
 pub type Address = Bytes20;
+
+/// Alias representing a storage key.
 pub type StorageKey = Bytes32;
+
+/// Alias representing a storage value.
 pub type StorageValue = Bytes32;
+
+/// Alias representing a log topic.
 pub type Topic = Bytes32;
+
+/// Alias representing a keccak or SHA-256 hash.
 pub type Hash = Bytes32;
+
+/// Alias representing a block's difficulty.
```

Comment:
Do we need alias in the text?

Reply:
Would you rather it be `Type`?

Reply:
I'm calling it a type is fair. That's how the user uses it.